pkg/operations/kubernetesupgrade/upgrader.go

```diff
@@ -231,6 +237,13 @@ func (ku *Upgrader) upgradeAgentPools(ctx context.Context) error {
 		if ku.DataModel.Properties.MasterProfile != nil {
 			isMasterManagedDisk = ku.DataModel.Properties.MasterProfile.IsManagedDisks()
 		}
+
+		if ku.DataModel.Properties.OrchestratorProfile.KubernetesConfig.LoadBalancerSku == api.StandardLoadBalancerSku {
+			err = transformer.NormalizeForK8sSLBScaling(ku.logger, templateMap)
```

Comment:
how will this affect AKS? 

scale.go is not used by AKS so `NormalizeForK8sSLBScaling` won't be called on AKS scales but this part of the upgrader is so AKS would go through this code when upgrading a VMAS agent (and same applies to VMSS agents)

Reply:
Not sure how AKS handles this today. @feiskyer @yangl900 might know?